kinto/config/kinto.tpl

```diff
@@ -65,22 +119,67 @@ multiauth.policies = basicauth
 # Set user 'account:admin' as the administrator.
 # kinto.account_write_principals = account:admin
 # kinto.account_read_principals = account:admin
+#
+# Kinto-portier authentication
+# https://github.com/Kinto/kinto-portier
+# multiauth.policies = portier
```

Comment:
Same doubt as for `kinto.includes`, should we repeat the setting in the file or just add a sentence?